app/models/version.rb

```diff
@@ -20,13 +20,26 @@ def numbers_must_increase
     errors.add(:number, message)
   end
 
+  def draft_and_live_versions
+    result = Queries::DraftAndLiveVersions.call(target, self.class.name.tableize)
```

Comment:
wouldn't really affect us unless we started changing table names, but this would be less brittle if `self.class.name.tableize` was `self.class.table_name`